docs/src/support/README.md

```diff
@@ -1,3 +1,70 @@
 # Getting Help #
```

Comment:
I've never seen having the `#`s either side of the title; did not know you could do that. I would be inclined to only have the leading `#`s to keep it consistent with our other documents.